lnwallet/channel.go

```diff
@@ -4331,6 +4331,16 @@ func (lc *LightningChannel) ReceiveHTLC(htlc *lnwire.UpdateAddHTLC) (uint64, err
 			"ID %d", htlc.ID, lc.remoteUpdateLog.htlcCounter)
 	}
 
+	// Clamp down on the number of HTLC's we can receive. It should never
+	// exceed our configurable MaxAcceptedHTLCs. The total number of HTLC's is
+	// the combination of uncommitted and committed HTLC's.
+	maxHTLCs := lc.channelState.LocalChanCfg.MaxAcceptedHtlcs
+	numHTLCs := lc.calcIncomingHtlcs(htlc.ID)
```

Comment:
Is this smart calculation now also applied on the side of the sender of the HTLC?

Reply:
It was, but turns out I was just mimicking `validateCommitmentSanity`